src/tree/transformer.ts

```diff
@@ -213,11 +213,11 @@ const getMetadata = (element: Source): ComponentMetadata => {
 };
 
 const getComponentDirectives = (metadata: ComponentMetadata): Array<string> => {
-  if (metadata == null || metadata.directives == null) {
+  if (metadata == null || metadata['directives'] == null) {
     return [];
```

Comment:
To add to that, this doesn't work for RC5 applications unless they still use the deprecated declarations. In RC6 they are gone altogether as you say. We should find some other way to determine which directives are invoked in the template. I'll drop this block out and put a TODO in here.